2005-05-06  Steven Brown  <swbrown@ucsd.edu>

	* Makefile.am: Add missing dist of autogen.sh.  Now disted
	images can be retooled.

	* scripts/Makefile.am: Remove incorrect dist of libtool.m4.
	This was blocking dists on various versions of libtool that
	don't create such a file.


diff -ur libsigcx-old/Makefile.am libsigcx/Makefile.am
--- libsigcx-old/Makefile.am	2005-05-06 19:48:48.000000000 -0700
+++ libsigcx/Makefile.am	2005-05-06 19:49:09.000000000 -0700
@@ -1,5 +1,6 @@
 SUBDIRS = sigcx tests @MAYBE_DOC@
 DIST_SUBDIRS = sigcx tests examples doc scripts
+EXTRA_DIST = autogen.sh
 
 ACLOCAL_AMFLAGS = -I scripts
 
diff -ur libsigcx-old/scripts/Makefile.am libsigcx/scripts/Makefile.am
--- libsigcx-old/scripts/Makefile.am	2005-05-06 19:48:48.000000000 -0700
+++ libsigcx/scripts/Makefile.am	2005-05-06 19:49:17.000000000 -0700
@@ -1 +1 @@
-EXTRA_DIST = macros.m4 pthread.m4 libtool.m4
+EXTRA_DIST = macros.m4 pthread.m4
